recipes-ni/ni-base-system-image-tests/files/known-permissions.csv

```diff
@@ -0,0 +1,2 @@
+Path,Mode,User,Group,Link Target
+/etc/fstab,-rw-r--r--,admin,administrators,
```

Comment:
Some other possible ideas (don't feed obligated to do all these):

 - everything in `/boot`, `/bin`, `/lib`, `/lib64`, `/sbin`, and `/usr` should be writable only by superuser
 - versioned kernels in `/boot`
 - home directories (don't bother with traversing their contents) should be mode 0700 (I think) and owned by their user
 - `/tmp` (or its link target) should be mode 1777

Might make more sense to do some of these if we had general predicates rather than explicit listings—it'd be annoying to maintain a CSV like this for some of these.

Reply:
I have added some of these. Feel free to look back over the csv again